### ISSUE TITLE
[Excel] taskpane panel apears blank

### ISSUE THREAD

**Issue by dinglicoding** [OfficeDev/office-js#1303]
## Expected Behavior

Load the plugin successfully.

## Current Behavior

The panel is blank when some of our clients open the plugin.

## Steps to Reproduce, or Live Example

We used Fiddler 4 to capture the network traffic and found the plugin did not request  http://nav.smartscreen.microsoft.com:443 , see attached zee.zip file.

On computers that can load the panel, we can see the request to http://nav.smartscreen.microsoft.com:443 .  Is this the cause of blank panel?

Also,can network connectivity/stability be the cause of this?   

## Your Environment

* Platform [PC desktop, Mac, iOS, Office on the web]: _PC desktop____
* Host [Excel, Word, PowerPoint, etc.]: ___excel___
* Office version number: __1908____
* Operating System: ___windows 1809___
* Browser (if using Office on the web): _office_____

## Useful logs

![image](https://user-images.githubusercontent.com/3963826/88677480-ebe3fd00-d11f-11ea-8f99-3c037a49dd96.png)
![image](https://user-images.githubusercontent.com/3963826/88677662-26e63080-d120-11ea-8aad-09e2c37990e7.png)
[zee.zip](https://github.com/OfficeDev/office-js/files/4988914/zee.zip)

**Comment by joshholat** [comment]
I don't have a ton of specifics yet, but we've also started to notice weird behavior along these lines with our add-on today. Basically not loading or only partially loading when it had been fine for quite a while previously. We've been debugging it all day trying to determine where the issue might be.

**Comment by keyur32** [comment]
(Updated title for clarity)

Hi there, thank you for the report and sorry for the convenience. We will need some additional information to debug this and follow up offline. 

If both of you can, please send a frown from within the Office application, as detailed here: https://support.microsoft.com/en-us/office/how-do-i-give-feedback-on-microsoft-office-2b102d44-b43f-4dd2-9ff4-23cf144cfb11. 

In the description, please be sure to indicate "To Office Platform team: Following up on issue https://github.com/OfficeDev/office-js/issues/1303.  Please contact me for further information".

**Comment by bdentino** [comment]
We have been having similar issues as well. After a lot of debugging, I believe in our case the issue is related to buggy behavior when using OfficeRuntime.storage.setItem/getItem. This issue only occurs on Windows, and we are using a shared runtime, so it's running in the IE 11 browser context. Essentially, in the onReady callback, we make concurrent requests to OfficeRuntime.storage.setItem and wait for them to complete. Occasionally, one of those promises simply never resolves (no error, no callback, just swallows the request), so all further execution is essentially blocked and the js we have to enable components in the taskpane never runs (thus the taskpane appearing to be blank). 

@dinglicoding @joshholat Are either of you using OfficeRuntime.storage in your `onReady` callback? If so, have you verified that those calls actually complete?

**Comment by dinglicoding** [comment]
@bdentino  thanks for your quick reply.

we have not use OfficeRuntime.storage.

**Comment by joshholat** [comment]
@bdentino We actually are seeing very similar behavior with regards to promises never resolving. We recently "fixed" the issue we were seeing with the diff below. Essentially, we realized that the `await context.sync()` towards the end of the function was simply hanging and never returning. I had a suspicion that those two operations were somehow conflicting with each other so figured I'd split them in to separate context syncs. That resolved our issue in that neither of those syncs hangs forever anymore.

This particular block of code had been this way for well over a month but only recently started causing issues and only on Windows Excel (e.g. IE like you).

![Screen Shot 2020-07-30 at 5 43 23 AM](https://user-images.githubusercontent.com/477603/88908237-9b57c700-d227-11ea-8ad7-7308a550576c.png)

Also, completely unrelated, I noticed you were at Finbox. I believe I met you or one of your other founders at one point. Kiran and I are good friends.
